content/api/hub/plugin_spin_cron_trigger.md

```diff
@@ -0,0 +1,66 @@
+title = "Spin Cron Trigger"
+template = "render_hub_content_body"
+date = "2022-10-15T00:22:56Z"
+content-type = "text/plain"
+tags = ["trigger", "rust"]
+
+[extra]
+author = "Fermyon"
+type = "hub_document"
+category = "Plugin"
+language = "Rust"
+created_at = "2023-12-05T00:22:56Z"
+last_updated = "2023-12-05T00:22:56Z"
+spin_version = “>=v2.0.1"
+summary = "An experimental cron trigger for Spin apps"
+url = "https://github.com/fermyon/spin-trigger-cron"
+keywords = "trigger, cron"
+
+---
+
+An experimental plugin that enables cron trigger for Spin applications. 
+
+### Prerequisites
+
+* [Rust](https://www.rust-lang.org/tools/install)
+* `pluginify` plugin (see below)
+
+```bash
+spin plugins install --url https://github.com/itowlson/spin-pluginify/releases/download/canary/pluginify.json
+```
+## Installing Template
```

Comment:
There seems to be a section missing on installing the actual trigger, or am I being more than usually blind here?

Reply:
Good catch! I would just replace the installing pluginify with the step to install the trigger plugin.